docs/core/porting/index.md

```diff
@@ -13,35 +13,96 @@ You might have code that currently runs on the .NET Framework that you're intere
+
+We recommend you use the following process when porting your project to .NET Core. Each of these steps introduces potential places for behavior changes, so ensure that you adequately test your library or application before continuing on to later steps. The first steps are to get your project ready for a switch to .NET Standard or .NET Core. If you have unit tests, it's best to convert them first so that you can continue testing changes in the product you're working on. Because porting to .NET Core is such a significant change to your codebase, it's highly recommended to port your test projects so that you can run tests as you port your code over. MSTest, xUnit, and NUnit all work on .NET Core.
+
+## Getting started
+
+The following tools will be used throughout the process:
+
+- Visual Studio 2019
+- Download [.NET Portability Analyzer](../../standard/analyzers/portability-analyzer.md)
+- _Optional_ Install [dotnet try-convert](https://github.com/dotnet/try-convert)
+
+## Porting a solution
+
+When there are multiple projects in a solution the porting can seem more complicated because you must address projects in a specific order. The conversion process should be a bottom-up approach, where the projects with no dependencies on other projects in the solution are converted first, and continue up through the whole solution.
+
+In order to identify the order projects should be migrated, you can use the following tools:
+
+- [Dependency Diagrams in Visual Studio](/visualstudio/modeling/create-layer-diagrams-from-your-code) can create a directed graph of the code in a solution.
+- Run `msbuild _SolutionPath_ /t:GenerateRestoreGraphFile /p:RestoreGraphOutputPath=graph.dg.json` to generate a json document that includes list of project references.
+
+## Per project steps
+
 We recommend you use the following process when porting your project to .NET Core:
 
+1. Convert all of your `packages.config` dependencies to the [PackageReference](/nuget/consume-packages/package-references-in-project-files) format with the [conversion tool in Visual Studio](/nuget/consume-packages/migrate-packages-config-to-package-reference).
```

Comment:
Note that this only works for old-style, unflavored C# and VB projects. ASP.NET, F#, and C++/CLI don't support this (C++/CLI doesn't support PackageReference at all yet, but it is planned for 16.8). Same goes for Xamarin, IIRC - which will be relevant for .NET 5.